code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2562,11 +2562,12 @@
 	color = "#9A6750" //RGB: 154, 103, 80
 	taste_description = "inner peace"
 	penetrates_skin = NONE
+	var/datum/disease/transformation/gondola_disease = /datum/disease/transformation/gondola
 
 /datum/reagent/gondola_mutation_toxin/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume, show_message = TRUE, touch_protection = 0)
 	. = ..()
 	if((methods & (PATCH|INGEST|INJECT)) || ((methods & VAPOR) && prob(min(reac_volume,100)*(1 - touch_protection))))
-		exposed_mob.ForceContractDisease(new /datum/disease/transformation/gondola(), FALSE, TRUE)
+		exposed_mob.ForceContractDisease(new gondola_disease(), FALSE, TRUE)
```

Comment:
```suggestion
		exposed_mob.ForceContractDisease(new gondola_disease, FALSE, TRUE)
```

no need for the parentheses in a `new` call if we aren't feeding it any arguments